new-employees/Secrets-in-the-office.md

```diff
@@ -0,0 +1,84 @@
+# The Secrets of the Creuna Office #
+
+There is a lot of little secrets hidden around in the office. 
+Some of those spots have taken your co-workers years to find.
+You will get alot of those reveal for you in this list. <br>
+Feel free to add your own.
+
+
+## Hardware and Licenses ##
+
+Do you have a wish for special hardware or long for a license for a nice program? All that and more can be granted by the IT department. Send an email to it@creuna.no or just stroll in and ask. Remeber, ask nicely. 
+The IT department is located in the other end of the building from the entrence. Behind the second mezzanine.
+
+## Communication ##
+
+Of course everyone have an email. Of course no one sends emails (exept for call to meetings). Most of all internal comunication is handled on [Slack](https://slack.com/). It is highly recomanded to download both on desktop and cellphone. Join the creuna-no.slack.com team by signing in with your Creuna username. Channels that might be good to join:
+* #creuna_fredagspils
+* #frontend
+* #developers
+* #creuna-github
+* #random
+<!--more channels?-->
+
+Of course you also need to join the #team-ui, but this is a locked channel som you will need an invitation from one of your fellow team members, or Pappa himself. There is alot of not-so-serius stuff going on on Slack, but dont think you can turn of all notifications and ignore it. Alot of important information is also passed on Slack. If you don't pay attention to Slack you might find out that you don't know where that important meeting moved or that someone can help you with a problem. Also, you might miss cake time. 
+
+You should also join [workplace](https://creuna.facebook.com/). Sign in with your Creuna username and password. Take a bit of time and find good groups to follow. Some tips are:
+* NO Creuna Norway
+* ALL Unit4 in Creuna
+* NO KAPASITET
+* NO Oslo Tech - Faggruppe
+* NO Nytt og nyttig fra HR og Admin. Norway
+<!--more groups?-->
+
+## Time registration ##
+All employees in Creua will have to endure registration of their work hours in Unit4. 
+The experience can be made better with this Chrome [plugin](https://github.com/Creuna-Oslo/Unit4Enhancements). 
+There is some good instruction videos in the workplace [Unit4 Group](https://creuna.facebook.com/groups/1056335684500691/videos/). The information for registrate your hours for diffrent projects is usually given by the project leader. 
+
+## Access cards and the alarm ##
+You have most likely got an access card and a code for that card. Remember that code. You will need that code if you arrive early (before 8.00)or work late(after 16.00).  After 23.00 and in the weekends the alarm is active in the office. If you use your card at the card reader the alarm will be off for 3 hours. If the alarm starts you will have to call Nokas on 02580 to deactivate it. You will have to give the password to Nokas. By obvious reasons I will not print the password here.. 
```

Comment:
Should we really have details about  alarm system times available publicly?

Reply:
No, we should not.

Reply:
Added it on request. But now it is gone.